bin/pygrb/pycbc_pygrb_pp_workflow

```diff
@@ -24,165 +24,324 @@
 Workflow generator to run pygrb offline post-processing.
 """
 
-from __future__ import division
-
+import sys
+import socket
 import logging
 import argparse
 import os
 import pycbc.version
+from pycbc import init_logging
 import pycbc.workflow as _workflow
-from pycbc.workflow.plotting import PlotExecutable, excludestr, requirestr
-from pycbc.results import create_versioning_page, layout, pygrb_plotting_utils
+from pycbc.workflow.plotting import PlotExecutable
+from pycbc.results import create_versioning_page, layout, pygrb_postprocessing_utils
 from pycbc.results.versioning import save_fig_with_metadata
+from pycbc.workflow import pegasus_workflow as wdax
+from Pegasus import DAX3 as dax
```

Comment:
This is another "Old code needs updating before it can work on master" problem. We moved to pegasus5 as part of the python3 transition, but you are calling the old `DAX3` module. While this does still exist in pegasus, it will soon be removed. We are now using `Pegasus.api` as recommended. As `pegasus_workflow` has moved over to this, this code will be broken in many places. I'm reluctant to use the main pycbc repo. as a repository for broken code that is being promised to be fixed later, so it would be good to demonstrate that this branch can actually do a GRB PP workflow?

Reply:
I've commented on this in a summary message below (March 4, 2022).